### ISSUE TITLE
Fatal error: Class 'ResourceBundle' not found

### ISSUE THREAD

**Issue by Raulken** [FriendsOfSymfony/FOSFacebookBundle#194]
i have this error when try to edit my profile...
this error i have after install fosfacebookbundle...
i read in web that i need to intl extension in my webserver....but i not have installed :(
exists another solution? for example edit htaccess?

thx

**Comment by Baachi** [comment]
Editing a `.htaccess` dosn't help ;)

You must write your own [DataTransformers](https://github.com/symfony/symfony/tree/master/src/Symfony/Component/Form/Extension/Core/DataTransformer). 
But i would suggest, you should install the `intl` extension.

```
$ sudo apt-get install php5-intl
```

**Comment by Raulken** [comment]
yes  in locale i have intl extension....but in webserver i haven't :(((((
i can't re.buy another server....

**Comment by Baachi** [comment]
Can you paste your autoload.php? 
Symfony should fallback to the Locale component if intl is not available.

**Comment by Raulken** [comment]
<?php

use Doctrine\Common\Annotations\AnnotationRegistry;

$loader = require **DIR**.'/../vendor/autoload.php';
$loader->add("Application", **DIR**);
// intl
if (!function_exists('intl_get_error_code')) {
    require_once **DIR**.'/../vendor/symfony/symfony/src/Symfony/Component/Locale/Resources/stubs/functions.php';

```
$loader->add('', __DIR__.'/../vendor/symfony/symfony/src/Symfony/Component/Locale/Resources/stubs');
```

}

AnnotationRegistry::registerLoader(array($loader, 'loadClass'));

return $loader;

**Comment by stof** [comment]
@Raulken The ResourceBundle class does not have a stub implementation yet

**Comment by Raulken** [comment]
sorry stof i not understand.....i'm italian and also with google translate i not understand
can u repeat with other words please?

**Comment by Raulken** [comment]
@stof i must have intl installed?
not exists another solution?

**Comment by Joey92** [comment]
Install php5-intl.

If you're using ubuntu or so, "sudo apt-get install php5-intl"
Or just uncomment the intl extension in you php.ini if your're using windows.
